tests/core_tests/test_pipeline_configuration.py

```diff
@@ -171,6 +174,45 @@ def setUp(self):
         super(TestConfigLocations, self).setUp({"primary_root_name": "primary"})
         self._project = self.mockgun.create("Project", {"name": "config_locations_test"})
 
+    def test_token_resolution_with_installed_configuration(self):
```

Comment:
This is the only test that actually tests the tokens subtitution. It does so for both configuration types and with every token, descriptor and path types.